deps/wazuh_testing/wazuh_testing/logcollector.py

```diff
@@ -760,4 +761,6 @@ def format_macos_message_pattern(process_name, message, type='log', subsystem=No
         elif type == 'activity':
             macos_message = f"{process_name}\[\d+\]: Created Activity ID.* Description: {message}"
 
+    assert macos_message is not None
```

Comment:
It would be good to put a message indicating the reason when the assert fails

Reply:
https://github.com/wazuh/wazuh-qa/pull/1478/commits/ed0810cdfd21063239284355eaef29c34bbbdd8e